syntax = "proto2";

import "common/proto/geometry.proto";

package interface.route;

message Route {
    optional interface.geometry.Point2D start_point = 1;
    optional interface.geometry.Point2D end_point = 2;
    // Sampled points in the route from start_point to end_point.
    // The distance between two adjacent should be less than 4m.
    // The distance between the first point in the route_point and start_point should be less than 4m.
    // The distance between the last point in the route_point and end_point should be less than 4m.
    repeated interface.geometry.Point2D route_point = 3;
}
